package-scripts/datadog-agent/postinst

```diff
@@ -3,6 +3,8 @@
 CONFIG_DIR=/etc/dd-agent
 LOG_DIR=/var/log/datadog
 RUN_DIR=/opt/datadog-agent/run
+JMX_STATUS_PYTHON=/tmp/jmx_status_python.yaml
+JMX_STATUS=/tmp/jmx_status.yaml
```

Comment:
The only issue I see is for customers with some weird configuration about their temp directory. (since we're calling ` tempfile.gettempdir()`: https://github.com/DataDog/dd-agent/blob/6eb139010510389c4cf97a1598a93152e8320d2a/config.py#L898)
I remember seing a customer with specific `/tmp` directories for each user. (something like `/tmp/{id}`)
Not sure if there is a correct solution for it though.